interesting-drink/interesting-drink.js

```diff
@@ -0,0 +1,38 @@
+// https://codeforces.com/problemset/problem/706/B
+
+function calculateShops(n, prices, days, coins) {
+    if (n < 1 || n > 100) {
+        return "Wrong number of shops";
+    }
+    for (let i of prices) {
+        if (i < 1 || i > Math.pow(10, 5)) {
+            return "Wrong price of the bottle";
+        }
+    }
+    if (days < 1 || days > Math.pow(10, 5)) {
+        return "Wrong number of days";
+    }
+    for (let i of coins) {
+        if (i < 1 || i > Math.pow(10, 5)) {
+            return "Wrong number of coins, that Vasili can spend";
+        }
+    }
+
+    let result = [];
+    for (let i = 0; i < days; i++) {
```

Comment:
Pomysł czy da się to zrobić bez zagnieżdżania pętli. Jak nic nie wymyślisz do zajęć to zostanie tak jak jest